packages/plugin-table/src/TableCellNode.ts

```diff
@@ -162,7 +157,10 @@ export class TableCellNode extends ContainerNode {
         this.__managedCells.add(cell);
 
         // Copy the manager's attributes and properties.
-        cell.attributes = { ...this.attributes };
+        const attributes = this.modifiers.get(Attributes);
+        if (attributes) {
+            cell.modifiers.replace(Attributes, attributes.clone());
+        }
```

Comment:
Actually, this is weird. I just realized it now but it might apply in other places as well. It's weird to only look for attributes here. Attributes is something that is proper to xml. I don't think you should manipulate them outside of xml parsers and renderers. Shouldn't you clone all modifiers here ?

Reply:
I believe you're actually right. Any other places where you think it might be the case?

Reply:
I think basically everywhere you would access `Attributes` outside of parsers and renderers... ^^'

Reply:
Well there was one: in `applyPreStyle`.